Taskfile.yml

```
@@ -4,6 +4,25 @@ includes:
   dist: ./DistTasks.yml
 
 tasks:
+  docs:publish:
+    desc: Use Mike to build and push versioned docs
+    deps:
+      - docs:gen:commands
+    cmds:
+      - poetry run mike deploy --update-aliases --push --remote {{.DOCS_REMOTE}} {{.DOCS_VERSION}} {{.DOCS_ALIAS}}
+
+  docs:gen:commands:
+    desc: Generate command reference files
+    dir: ./docsgen
+    cmds:
+      # docs will generate examples using os.Args[0] so we need to call
+      # the generator `arduino-cli`
+      - go build -o {{.PROJECT_NAME}}
```

```suggestion
      - go build -o {{.PROJECT_NAME}}{{exeExt}}
```
This is necessary to allow Windows users to run the command in order to test the website locally.
https://taskfile.dev/#/usage?id=gos-template-engine

---

I did not know about that trick! Thanks!